README.md

```diff
@@ -148,7 +149,33 @@ $ eregs build_from CFR-2012-title12-vol8-part1004.xml 12
 
 Here ```12``` is the CFR title number (in our case, for "Banks and Banking").
 
-Running the command will generate four folders, ```regulation```,
+
+#### Using `pipeline`
+```bash
+$ eregs pipeline title part an/output/directory
+```
+or
+```bash
+$ eregs pipeline title part https://yourserver/
+```
+
+Example:
+```bash
+$ eregs pipeline 27 447 /path/to/reg
```

Comment:
Something like `/output/path` might make this clearer -- this is where the output would be written